src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -105,61 +110,82 @@ namespace OpenRCT2::Ui::Windows
 
     static char _filter_string[MAX_PATH];
 
-#define _FILTER_ALL ((_filter_flags & FILTER_ALL) == FILTER_ALL)
-#define _FILTER_RCT1 (_filter_flags & FILTER_RCT1)
-#define _FILTER_AA (_filter_flags & FILTER_AA)
-#define _FILTER_LL (_filter_flags & FILTER_LL)
-#define _FILTER_RCT2 (_filter_flags & FILTER_RCT2)
-#define _FILTER_WW (_filter_flags & FILTER_WW)
-#define _FILTER_TT (_filter_flags & FILTER_TT)
-#define _FILTER_OO (_filter_flags & FILTER_OO)
-#define _FILTER_CUSTOM (_filter_flags & FILTER_CUSTOM)
-#define _FILTER_SELECTED (_filter_flags & FILTER_SELECTED)
-#define _FILTER_NONSELECTED (_filter_flags & FILTER_NONSELECTED)
+    static bool isFilterActive(const uint16_t filter)
+    {
+        return (_filter_flags & filter) == filter;
+    }
 
     static constexpr StringId WINDOW_TITLE = STR_OBJECT_SELECTION;
     static constexpr int32_t WH = 400;
-    static constexpr int32_t WW = 755;
+    static constexpr int32_t WW = 600;
+    static constexpr auto kFilterWidth = 150;
```

Comment:
```suggestion
    static constexpr auto kFilterWidth = 175;
```

@Gymnasiast Do you think this will provide enough room?

Reply:
I would personally be inclined to go for 200. If that doesn’t sound right to you, we could go with 175 and see what the translators say.

Reply:
Hmm, I just think 200px will look very empty for 'All sources shown'. I'm inclined to go with 175px and expand further if needed then.